spec/components/schools/participants/status_spec.rb

```diff
@@ -0,0 +1,293 @@
+  component { described_class.new(participant_profile: profile) }
+
+  context "when an email has been sent" do
+    let(:profile) { create(:participant_profile, :ecf, :email_sent) }
+
+    it "displayed the details required content" do
+      expect(rendered).to have_content I18n.t "schools.participants.status.details_required.header"
+      I18n.t("schools.participants.status.details_required.content").each do |content|
+        expect(rendered).to have_content content
+      end
+    end
+  end
+
+  context "when an email bounced" do
+    let(:profile) { create(:participant_profile, :ecf, :email_bounced) }
+
+    it "displays the request for details failed content" do
+      expect(rendered).to have_content I18n.t "schools.participants.status.request_for_details_failed.header"
+      expect(rendered).to have_content I18n.t "schools.participants.status.request_for_details_failed.content"
+    end
+  end
+
+  context "when no email has been sent" do
+    let(:profile) { create(:participant_profile, :ecf) }
+
+    it "displays the request to be sent content" do
+      expect(rendered).to have_content I18n.t "schools.participants.status.request_to_be_sent.header"
+      expect(rendered).to have_content I18n.t "schools.participants.status.request_to_be_sent.content"
+    end
+  end
+
+  context "when the participant is doing FIP" do
+    let(:school_cohort) { create(:school_cohort, :fip) }
+    context "when the participant is an ECT" do
+      let(:profile) { create(:participant_profile, :ect, school_cohort: school_cohort) }
+      let!(:validation_data) { create(:ecf_participant_validation_data, participant_profile: profile) }
+
+      context "when the participant is eligible" do
+        let!(:eligibility) { create(:ecf_participant_eligibility, :eligible, participant_profile: profile) }
+
+        it "displays the eligible fip no partner content" do
+          expect(rendered).to have_content I18n.t "schools.participants.status.eligible_fip_no_partner.header"
+          expect(rendered).to have_content I18n.t "schools.participants.status.eligible_fip_no_partner.content"
+        end
+
+        context "when the school is in a partnership" do
+          let!(:partnership) { create(:partnership, school: school_cohort.school, cohort: school_cohort.cohort) }
+
+          it "displays the eligible fip content" do
+            expect(rendered).to have_content I18n.t "schools.participants.status.eligible_fip.header"
+            expect(rendered).to have_content I18n.t "schools.participants.status.eligible_fip.content"
+          end
+        end
+      end
+
+      context "when the participant has no QTS" do
+        let!(:eligibility) { create(:ecf_participant_eligibility, qts: false, participant_profile: profile) }
+
+        it "displays the fip ect no qts content" do
+          expect(rendered).to have_content I18n.t "schools.participants.status.fip_ect_no_qts.header"
```

Comment:
Nitpicking again "FIP no QTS"

Reply:
Ah, I left these lowercase to match the I18n key - the content is `:fip_ect_no_qts` : https://github.com/DFE-Digital/early-careers-framework/blob/CPDRP-1015-ParticipantDetails/config/locales/schools/participants.yml#L40